Platform/CoffeelakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1602,48 +1602,53 @@ SaveNvsData (
   if (EFI_ERROR(Status)) {
```

Comment:
Need get MRC region size and check if (NvsSize < Length)  before really saving NVS data.

Reply:
Updated new patch